### PR TITLE
Schema uses lower cases for columns and minor improve on error messages

### PR DESCRIPTION
**Pull request by yigress**

### CODE REVIEW

**Comment by jphalip** [comment]
Does that fix a potential bug? If so, could some tests be added?

**Comment by yigress** [comment]
no, this won't fix the batch commit error seen before. i can't reproduce that issue and thus add more error message.
the schema lower case change is for spark usage when spark tables are created with upper case. 
hard to add integration tests right now for above changes.

File: connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/arrow/ArrowRecordReader.java [hunk 1]
```diff
@@ -56,9 +56,14 @@ private Object serializeRow(VectorSchemaRoot schemaRoot, int rowId) {
     Object[] row = new Object[columnNames.size()];
     for (int i = 0; i < numColumnsInSchemaRoot; i++) {
       FieldVector fieldVector = schemaRoot.getVector(i);
-      int colIndex = columnNames.indexOf(fieldVector.getName());
+      String fieldName = fieldVector.getName();
+      int colIndex = columnNames.indexOf(fieldName);
+      if (colIndex == -1) {
+        throw new RuntimeException(
+            "Unable to find colum " + fieldName + " in columns " + columnNames);
```
Comment (functicons) [inline connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/arrow/ArrowRecordReader.java:63]: typo: colum

**Review by functicons — APPROVED** [review]
LGTM

File: connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/arrow/ArrowRecordReader.java (hunk 1 above)
Comment (yigress) [reply connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/arrow/ArrowRecordReader.java]: fixed typo

**Pull request merged**